RealEstate/3 reports.sql

```diff
@@ -0,0 +1,21 @@
+--1) I want a report of all the houses sorted by the block and then by the town/city
```

Comment:
see my comment on the columns that should be used for the reports

Reply:
> address, town, Realtor, seller, **sell date**, buyer, and **purchase date**

Looks like `DateSold` is repeated, so I only returned it once. Let me know if you intended a different column!